rero_ils/acquisition/proxies.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+#
+# RERO ILS
+# Copyright (C) 2021 RERO
+# Copyright (C) 2021 UCLouvain
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU Affero General Public License as published by
+# the Free Software Foundation, version 3 of the License.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+# GNU Affero General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with this program. If not, see <http://www.gnu.org/licenses/>.
+
+"""Helper proxy to the state object."""
```

Comment:
I guess that does mean something. But I can't understand it. ^^

Reply:
Done

Reply:
> For the commit component: `acquisitions` with an 's'.
> 
> By the way, should the directory `rero_ils/acquisition` be `rero_ils/acquisitions`? Or is it to avoid confusion with `rero_ils/modules/acquisitions`?

I understand, but I consider it more like an acquisition module.